src/CacheManager.Serialization.ProtoBuf/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("CacheManager.Serialization.ProtoBuf")]
+[assembly: AssemblyTrademark("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("85cd8bc7-3e72-422e-af91-6fdf2133e155")]
```

Comment:
In xproj, I use the AssemblyInfo.cs only for setting CLSCompliant(true) if possible. If it is not possible, just remove the file as it will not be used anyways.